lib/node_modules/@stdlib/_tools/eslint/rules/require-last-path-relative/lib/main.js

```diff
@@ -0,0 +1,124 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var startsWith = require( '@stdlib/string/starts-with' );
+
+
+// FUNCTIONS //
+
+/**
+* Checks if a require path is relative.
+*
+* @private
+* @param {string} path - require path
+* @returns {boolean} boolean indicating if path is relative
+*/
+function isRelativePath( path ) {
+	return (
+		startsWith( path, './' ) ||
+		startsWith( path, '/' ) ||
+		startsWith( path, '../' )
```

Comment:
It may also be good to have another ESLint rule which disallow relative paths of the form `../`, as, I believe, we use `./` almost exclusively.

Reply:
We already have it: `stdlib/require-leading-slash`.